dev/openchpl_data-model.sql

```diff
@@ -2976,6 +2976,36 @@ CREATE TABLE openchpl.broken_surveillance_rules
     CONSTRAINT broken_surveillance_rules_id_pk PRIMARY KEY (id)
 );
 
+CREATE TABLE openchpl.file_type
+(
+    file_type_id bigserial NOT NULL,
+    name text NOT NULL,
+    description text,
+	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+    last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+    last_modified_user bigint NOT NULL,
+    deleted boolean NOT NULL DEFAULT false,
+    CONSTRAINT file_type_pk PRIMARY KEY (file_type_id)
+);
+
+CREATE TABLE openchpl.files
```

Comment:
I think it would be more in line with sql convention to not have a plural table name